loadbalancer_private.tf

```diff
@@ -159,6 +159,15 @@ resource "aws_lb_listener" "quortex_private_tls" {
   }
 }
 
+# Provides Load Balancer Listener Certificate resources.
+# These resources are for additional certificates and does not replace the default certificate on the listener.
+resource "aws_lb_listener_certificate" "quortex_private" {
+  for_each = var.load_balancer_private_expose_https ? (length(var.load_balancer_private_app_backend_ports) > 0 ? var.load_balancer_private_additional_certs_arns : []) : []
```

Comment:
I'm not sure to understand what you are doing, are there two ternary here ?
Couldn't you write it like this (or did I misunderstood what you were doing) ?
```
  for_each = var.load_balancer_private_expose_https && length(var.load_balancer_private_app_backend_ports) > 0 ? var.load_balancer_private_additional_certs_arns : []
```

Reply:
Yes, it's more readable like that... fixed

Reply:
I still does not understand, why do you need load_balancer_private_app_backend_ports to set additional certificates ?

Reply:
It's the actual condition for alb HTTPS listeners creation.